oml/functional/metrics.py

```diff
@@ -153,10 +152,11 @@ def reduce_metrics(metrics_to_reduce: TMetricsDict) -> TMetricsDict:
 
 
 def apply_mask_to_ignore(
-    distances: torch.Tensor, mask_gt: torch.Tensor, mask_to_ignore: torch.Tensor
+    distances: torch.Tensor, mask_gt: Optional[torch.Tensor], mask_to_ignore: torch.Tensor
 ) -> Tuple[torch.Tensor, torch.Tensor]:
     distances[mask_to_ignore] = float("inf")
-    mask_gt[mask_to_ignore] = False
+    if mask_gt is not None:
```

Comment:
The such modification adds the possibility to make new errors in the future. And you use this modification to reduce 1 line of code in the new test. Worth it?

Reply:
ok